rhobot.go

```diff
@@ -57,6 +57,16 @@ func main() {
 		Value: "",
 		Usage: "yaml file containing email distribution list",
 	}
+	hcSchemaFlag := cli.StringFlag{
+		Name:  "hcschema",
```

Comment:
renamed the variables in 0abe75df7abb0c619d32ee9a403f99a60be18a4d 
these flags are only looked for during the healthchecks command